src/ExportPro.Common/ExportPro.Common.Shared/Library/SuccessResponseT.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Net;
+
+
+namespace ExportPro.Common.Shared.Library;
+
+public class SuccessApiServiceResponse<T> : BaseResponseT<T>
```

Comment:
please keep consistent between naming. we can name generic method and non generic with the same name